tools/Zig.defs

```diff
@@ -39,4 +39,6 @@ endif
 
 # Convert cortex-xxx/sifive-exx to cortex_xxx/sifive_exx
 
+ifneq ($(LLVM_CPUTYPE),)
 ZIGFLAGS += -mcpu $(subst -,_,$(LLVM_CPUTYPE))
+endif
```

Comment:
Any idea why only Zig does this kind of substitution for LLVM?

Reply:
No idea, it was there before and I got empty string during local test, so I guarded it to move ahead.

Reply:
@acassis it seems that "sim/wamr" is blocking the CI checks, will push again later until it is resolved.